esmvaltool/recipes/recipe_OceanBGC.yml

```diff
@@ -0,0 +1,132 @@
+###############################################################################
```

Comment:
Please use the tool `yamllint` to check that your yaml file looks as good as possible, you can ignore any comments about lines too long or too much whitespace if you think your layout is nicer than what yamllint suggests.